docs/en/latest/plugins/skywalking.md

```diff
@@ -78,20 +79,20 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
 }'
 ```
 
-You also can complete the above operation through the web interface, first add a route, then add skywalking plugin:
+You also can complete the above operation through the web interface, first add a route, then add SkyWalking plugin:
 
 ![ ](../../../assets/images/plugin/skywalking-1.png)
 
-## How to set endpoint
+## How to Set Endpoint
```

Comment:
why change `set endpoint` to `Set Endpoint`? This title looks strange.

Reply:
The level two ## titles are not using a consistent capitalization, half of them are all caps while the others aren't, so I was trying to make them match. The same goes to the endpoint one below.

Reply:
@tzssangglass Would you kindly suggest that I simply revert the changes or make the other ones consistent with the `Cap lower` style?